### ISSUE TITLE
Readme: usage is wrong

### ISSUE THREAD

**Issue by mpachas** [romcal/romcal-api#7]
It says:
const romcalMiddleware = require('romcal-api').expressMiddleware(romcalConfig); 
but it should be:
const romcalMiddleware = require('romcal-api').romcalExpressMiddleware(romcalConfig);

**Comment by emagnier** [comment]
Good point. Thank you for your feedback. This project is currently on standby (for lack of time), and may only evolve when the next major version of romcal will be released.
